simpleflow/canvas.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, func, *args, **kwargs):
 
     def submit(self, executor):
```

Comment:
(minor, can be refactored later) Wouldn't it be better to pass the workflow directly instead?